### PR TITLE
Add getter methods for condition

### PR DESCRIPTION
**Pull request by ryoqun**
hey, thanks for creating this crate.

it's concise and customizable. also, it's what I exactly wanted instead of writing my own. ):

hope this small improvement makes sense. 

as a context, this is my use case: 
https://github.com/ryoqun/solana/blob/ab5b796e8bda0f13678ed109aa7bac756cfc5b57/core/src/banking_trace.rs#L317

### CODE REVIEW

**Comment by ryoqun** [comment]
@kevinhoffman friendly ping. :) (btw, i have another commit to upstream.)

File: src/lib.rs [hunk 1]
```diff
@@ -222,6 +222,11 @@ where
         self.open_writer_if_needed()
     }
 
+    /// Retrieves the rolling condition, possibly to mutete its state dynamically.
```
Comment (apfitzge) [inline src/lib.rs]: Typo in "mutate":
```suggestion
    /// Retrieves the rolling condition, possibly to mutate its state dynamically.
```

**Comment by ryoqun** [comment]
@kevinhoffman just for completeness, i also added `.condition_ref()`: https://github.com/Axcient/rolling-file-rs/pull/3/commits/f3e97e6c337b3218fe17be4fa4edc4f80d313ae2

**Comment by kevinhoffman** [comment]
@ryoqun - thank you!

**Pull request merged**